### PR TITLE
FABN-1536 NodeSDK add getTransactionId

### PR DESCRIPTION
**Pull request by harrisob**
Add the getTransactionId to fabric-network.transaction

Signed-off-by: Bret Harrison <beharrison@nc.rr.com>

### CODE REVIEW

**Comment by bestbeforetoday** [comment]
Probably needs to be changed in the Java SDK too, to keep the programming model consistent.

File: fabric-network/types/index.d.ts [hunk 1]
```diff
@@ -99,6 +99,7 @@ export interface TransientMap {
 }
 export class Transaction {
 	getName(): string;
+	getTransactionId(): string;
```
Comment (bestbeforetoday) [inline fabric-network/types/index.d.ts:102]: Based on the code and tests, the return type should be `string | null`

**Review by bestbeforetoday — CHANGES_REQUESTED** [review]
TypeScript return type is not correct

File: fabric-network/types/index.d.ts (hunk 1 above)
Comment (harrisob) [reply fabric-network/types/index.d.ts]: fixed

**Review by bestbeforetoday — APPROVED** [review]

**Pull request merged**